packages/compass-home/package.json

```diff
@@ -81,6 +81,9 @@
     "sinon": "^8.1.1",
     "xvfb-maybe": "^0.2.1"
   },
+  "optionalPeerDependencies": {
+    "@electron/remote": "^2.0.8"
+  },
```

Comment:
Our depcheck scripts fail if this is added to `dependencies` and/or `peerDependencies`, unfortunately.

Reply:
What error are you seeing?

Reply:
@gribnoysup When in `dependencies` or `peerDependencies` only:

```
Errors found in the @mongodb-js/compass-home package.json dependencies description

Package `peerDependencies` should match `dependencies`
```

When in both:

```
Errors found in the @mongodb-js/compass-home package.json dependencies description

Following dependencies should not be in peerDependencies:

  @electron/remote
```

Reply:
Well, I’m still not entirely clear around the expected semantics here (esp. regarding why `peerDependencies` duplicates `dependencies`), so I’m not quite sure what the error message should actually be :) But yeah, for this problem, I guess moving it to dev deps works. I think I was just confused by the fact that the `devDependencies` list looked like actual dev dependencies, rather than dev dependencies + what ends up being bundled.